### DISCUSSION TITLE
Change Checkout default option from Scan tab to Copy

### DISCUSSION THREAD

**Discussion by sityware** [btcpayserver/btcpayserver#3800] [Ideas / Feature Requests]
I believe the checkout Copy tab is more user friendly as a first option. Scanning on my phone opens up an Email for example.

I would like a simple setting to change the default to be the Copy tab.

![image](https://user-images.githubusercontent.com/31103915/171201765-a2e7d3a3-4dc8-4280-8c6c-f73cd8e11307.png)

**Comment by bolatovumar** [comment]
> Scanning on my phone opens up an Email for example.

This depends on what apps you have installed on your phone and how your phone is set up. Also, the assumption with these QR codes generally that people scan them with a bitcoin wallet app open on their phone.

In any case this shouldn't be hard to implement if there is more interest in having this.
